test/command/suite/select/slices/function/sub_filter/pre_filter.expected

```diff
@@ -1,25 +0,0 @@
-table_create Users TABLE_HASH_KEY ShortText
```

Comment:
Note: `pre_filter.test` doesn't exist. we don't need `pre_filter.test`  or `pre_filter.test`.